gdp/small_batch/gdp_small_batch.py

```diff
@@ -35,25 +36,44 @@ def build_small_batch():
     m = pe.ConcreteModel()
 
     # Sets
-    m.i = pe.Set(initialize=['a', 'b'], doc='Set of produncts')  # Set of products, i = a, b
-    m.j = pe.Set(initialize=['mixer', 'reactor',
-                             'centrifuge'])  # Set of stages, j = mixer, reactor, centrifuge 
-    m.k = pe.RangeSet(NK)    # Set of potential number of parallel units, k = 1, 2, 3
+    m.i = pe.Set(
+        initialize=['a', 'b'], doc='Set of produncts'
+    )  # Set of products, i = a, b
+    m.j = pe.Set(
+        initialize=['mixer', 'reactor', 'centrifuge']
+    )  # Set of stages, j = mixer, reactor, centrifuge
+    m.k = pe.RangeSet(NK)  # Set of potential number of parallel units, k = 1, 2, 3
 
     # Parameters and Scalars
 
-    m.h = pe.Param(initialize=6000, doc='Horizon time [hr]')  # Horizon time  (available time) [hr]
-    m.vlow = pe.Param(initialize=250, doc='Lower bound for size of batch unit [L]')  # Lower bound for size of batch unit [L]
-    m.vupp = pe.Param(initialize=2500, doc='Upper bound for size of batch unit [L]')  # Upper bound for size of batch unit [L]
+    m.h = pe.Param(
+        initialize=6000, doc='Horizon time [hr]'
+    )  # Horizon time  (available time) [hr]
+    m.vlow = pe.Param(
+        initialize=250, doc='Lower bound for size of batch unit [L]'
+    )  # Lower bound for size of batch unit [L]
+    m.vupp = pe.Param(
+        initialize=2500, doc='Upper bound for size of batch unit [L]'
+    )  # Upper bound for size of batch unit [L]
 
     # Demand of product i
-    m.q = pe.Param(m.i, initialize={'a': 200000, 'b': 150000}, doc='Production rate of the product [kg]')
+    m.q = pe.Param(
+        m.i,
+        initialize={'a': 200000, 'b': 150000},
+        doc='Production rate of the product [kg]',
+    )
     # Cost coefficient for batch units
     m.alpha = pe.Param(
-        m.j, initialize={'mixer': 250, 'reactor': 500, 'centrifuge': 340}, doc='Cost coefficient for batch units [$/L^beta]')
+        m.j,
+        initialize={'mixer': 250, 'reactor': 500, 'centrifuge': 340},
+        doc='Cost coefficient for batch units [$/L^beta]',
```

Comment:
I think these units are wrong. Did you do a dimensional analysis or used a reference?